shell/src/main/java/alluxio/cli/validation/HdfsValidationTask.java

```diff
@@ -58,8 +58,8 @@ public TaskResult validate(Map<String, String> optionsMap) {
   }
 
   protected boolean shouldSkip() {
-    String scheme = new AlluxioURI(ServerConfiguration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS))
-        .getScheme();
+    String scheme = new AlluxioURI(InstancedConfiguration.defaults()
+                      .get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS)).getScheme();
```

Comment:
`InstancedConfiguration.defaults()` could be expensive. I recommend creating the configuration object on JVM startup and then passing that into this function.